schema/establishment.js

```diff
@@ -115,9 +115,9 @@ class Establishment extends BaseModel {
       .then(result => result.count);
   }
 
-  getPELH() {
+  getEstablishmentRole() {
```

Comment:
Not sure about the name of this function. Aren't all roles establishment roles?

Reply:
Got rid of the function.